src/interfaces/vault.ts

```diff
@@ -680,4 +688,29 @@ export class VaultInterface<T extends ChainId> extends ServiceInterface<T> {
     };
     return apy;
   }
+
+  /**
+   * Helper function to set the zapper properties on a vault's metadata
+   * @param metadataOverrides the vault metadata overrides
+   * @param vaultTokenMarketData the vault token market data
+   * @returns the updated metadata
+   */
+  mergeZapperProps(
+    metadataOverrides: VaultMetadataOverrides[],
+    vaultTokenMarketData: VaultTokenMarketData[]
+  ): VaultMetadataOverrides[] {
+    const vaultTokenMarketDataAddresses = new Set(vaultTokenMarketData.map(({ address }) => address.toUpperCase()));
+
+    return metadataOverrides.map((metadataOverride) => {
+      const isZappable = vaultTokenMarketDataAddresses.has(metadataOverride.address.toUpperCase());
```

Comment:
The address coming from `vaultTokenMarketData` and `metadataOverride` were the same, but different case. So I convert both to uppercase when doing the comparison.

Reply:
maybe use `import { getAddress } from "@ethersproject/address";` better?

With Zapper responses we always need to checksum addresses yes :S

Reply:
Good idea, forgot about that helper. Refactored with https://github.com/yearn/yearn-sdk/pull/276/commits/7cea4f3fc7d4bccb224c9e34cb0489362622f03f

Wondering if we should get both addresses or just the ones coming from zapper?

Reply:
in theory, all addresses on our data structures should be checksummed, unless we have a bug

Reply:
Should we trust that or be extra safe? I'm happy to remove the check from the commit above

Reply:
Lets be extra safe, its ok.